server.js

```diff
@@ -29,6 +29,32 @@ app.post('/project',function(req , res){
 		res.send(data)
 	})
 })
+
+//Routes for Tasks :)
+app.get('/tasks', function(req, res)
+{
+	db.Task.find({}, function(err, data)
+	{
+		if(err)
+		{
+			res.send(err);
+		}
+		res.status(200).send(data);
+	});
+});
+
+app.post('/tasks', function(req, res)
+{
+	db.addTask(req.body, function(err, data)
+	{
+		if(err)
+		{
```

Comment:
if statements should also have their opening bracket on the same line as the if keyword.
```
if (statement === true) {
  someFunctionWeWrote();
}
```

This way, we can see very easily that the `someFunctionWeWrote` function is being called _inside_ the if statement, and will only be called if the `statement` variable is true.